### ISSUE TITLE
"active" sessions

### ISSUE THREAD

**Issue by generalaggregator** [sienori/Tab-Session-Manager#383]
Wish: The one feature of the old session manager that I really miss is the active update of an opened session every so-many minutes and when it's closed (rather than a new session being generated each time).

**Comment by llewlem888** [comment]
Me too. This was the one thing I cared about most that mozilla killed with dropping XUL support. 

Off-topic: But we do have Single File Save now...   https://github.com/gildas-lormeau/SingleFile

**Comment by knezi** [comment]
So you mean tracking which window belongs to which session? This sounds a bit complicated to me.

For me it'd be enough just to be able to get a button next to each session for updating (going to the three dots & choosing only this window is somewhat tiresome). Or maybe being able to add an update button next to the search bar. When you click that, list of existing sessions appear, you choose one a this one gets overwritten.

Or maybe can you describe a bit more in details how exactly you want to treat that?

**Comment by Ghos3t** [comment]
Yes this is a very important feature, because when I re-open an old session I might end up opening new tabs or closing old ones etc. These changes are not made to the old session automatically but rather you have to manually go the session and click the three dot menu and select replace with current window or replace with current session. Also when you close the browser it creates a new session instead of updating the old session that was opened.

**Comment by tennox** [comment]
Check out [this PR](https://github.com/sienori/Tab-Session-Manager/pull/889)
